scripts/github-prerelease.ts

```diff
@@ -49,7 +49,7 @@ const createPreRelease = async (version: string) => {
 
   await Promise.all(
     platformSuffixes
-      .map(suffix => `${binaryNamePrefix}${suffix}`)
+      .map(suffix => `${binaryNamePrefix}${suffix}.tgz`)
```

Comment:
isn't the extension platform dependent?

Reply:
@edwardfoyle  can you please respond to this please? :)

Reply:
the platform specific part is included in ${suffix}. They are all compressed the same with .tgz extension